src/modules/datasets/ms_cm_exports.bicep

```diff
@@ -0,0 +1,63 @@
+@description('Required. The name of the parent Azure Data Factory.')
```

Comment:
Should we have a description of what this file is doing in the file itself?

Reply:
Good plan.  Done.